homework2/sass/header.scss

```diff
@@ -0,0 +1,343 @@
+                  flex-direction: column;
+                  align-items: center;
+
+                  &:after {
+                    content: "";
+                    width: 40px;
+                    height: 2px;
+                    background-color: $color-link;
+                    display: block;
+                    text-align: center;
+                    transition: all 0.3s ease 0s;
+                  }
+                }
+              }
+
+
+              &.get-skout-btn, &.login-btn {
+                button {
+                  @include tpl-button;
+                  box-shadow: none;
+                  margin: 200px 0 15px;
+                  &:hover{
+                    box-shadow: none;
+                  }
+                }
+              }
+
+              &.login-btn {
+                button {
+                  padding: 8px 22px;
+                  margin: 0 0 40px;
+                }
+              }
+            }
+          }
+
+          &.active {
+            left: 0;
+
+            &::before {
+              content: "";
+              position: fixed;
+              width: 100%;
+              top: 0;
+              left: 0;
+              height: 62px;
+              background: linear-gradient(45deg, #12C1E6, #01F973);
+              z-index: 2;
+              border: none;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .header-container-fixed {
+    position: fixed;
+    background-color: $color-primary;
+
+    .menu-icon {
+      span, &::before, &::after {
+        background-color: $color-secondary-text !important;
+      }
+    }
+
+    a {
+      color: $color-secondary-text !important;
+
+      &.current {
+        &:after {
+          background-color: $color-secondary-text !important;
+        }
+      }
+    }
+
+    .login-btn, .get-skout-btn {
+      button {
+        border-color: $color-secondary-text !important;
+        color: $color-secondary-text !important;
+
+        &:hover {
+          @include fontProp(14px, 24px, $color-primary);
+          color: $color-primary !important;
+          background-color: $color-secondary-text !important;
+        }
+      }
+    }
+  }
+
+
+  .header-content {
+    max-width: 1440px;
+    margin: 0 auto;
+
+    .header-title {
+      font-family: 'Actor', sans-serif;
+      @include fontProp(60px, 70px, $color-primary);
+      padding: 75px 30px 30px;
+      margin: 0 auto;
+      text-transform: uppercase;
+      text-align: center;
+    }
+
+    .header-text {
+      display: flex;
+      flex-direction: column;
+      padding: 0 30px 50px;
+
+      p {
+        padding-left: 20px;
+        @include fontProp(13px, 24px, $color-primary);
+        border-left: 1.5px solid $color-primary;
+        font-family: 'Aclonica', sans-serif;
+      }
+
+      button {
+        margin-top: 30px;
+        align-self: center;
+        @include tpl-button;
+        border: none;
+      }
+    }
+  }
+}
+
+@media (min-width: 1440px) {
```

Comment:
why you use only one media-query ? what about 1200, 768?

Reply:
Done 1440, 1200, 992, 768.